crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -383,4 +382,30 @@ impl AudioBackend for PwBackend {
 
         Ok(())
     }
+
+    fn release(&self, stream_id: u32) -> Result<()> {
+        println!("pipewire backend, release function");
+        self.thread_loop.lock();
+        let mut stream_states = self.stream_states.write().unwrap();
+        let stream_hash = self.stream_hash.read().unwrap();
+        let stream_listener = self.stream_listener.read().unwrap();
+
+        if let Some(stream) = stream_hash.get(&stream_id) {
+            stream.disconnect().expect("could not disconnect stream");
+        } else {
+            println!("stream not found for the given stream_id");
```

Comment:
What about using `error!` or `warn!`?

IIUC this error is not expected so, we don't want to fail the release in the middle and continue with the next steps, right?

At this point, is the `Result<()>` the right return type, or we should force all the backend to avoid failures without returning it?

Reply:
ok, I could use `error!`